src/main/resources/hudson/model/ExternalJob/newJobDetail.jelly

```diff
@@ -24,5 +24,5 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <div>
-  ${%body}
+  ${it.description}
```

Comment:
:bug: since the description includes HTML you must use

```xml
<j:out value="${it.description}"/>
```

to prevent inappropriate escaping.